### PR TITLE
#06 Feat: Create Link Item Component

### PR DESCRIPTION
**Pull request by mercihabam**
### Task Description _(❗️Required)_

Created a reusable link component for all links with arrow

### Type of Task

- [ ] fix
- [ ] chore
- [x] feat
- [ ] refactor
- [ ] perf
- [ ] docs

### Guidelines

- [x] Does your code follow ES6+ syntax?
- [x] Did you use arrow functional components?
- [x] Did you thoroughly review your code?
- [x] Have you checked if your code does not introduce conflicts in the main codebase?

### Ticket Link _(❗️Required)_

https://www.notion.so/apeunit/LinkItem-component-22e7c7c18dcb41e8a99a270dac9eec42

### CODE REVIEW

File: components/ui/LinkItem.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Image from 'next/image'
+
+const LinkItem = ({children}) => {
+  return (
+    <div className='flex items-center gap-1 group cursor-pointer'>
+        {children}
+        <div className="w-max group-hover:translate-x-1 group-hover:-translate-y-1 transition-all">
+            <Image width={14} height={14} src="/assets/icons/arrow-link.png" alt='arrow' />
```
Comment (Odile078) [inline components/ui/LinkItem.jsx]: You forgot to create an icon folder that contains this specific image that you used in this src.

**Review by Odile078 — COMMENTED** [review]
Looks good but there is a small change that you need to add.

**Review by Lily-Lilian — DISMISSED** [review]

File: components/ui/LinkItem.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Image from 'next/image'
+
+const LinkItem = ({children}) => {
+  return (
+    <div className='flex items-center gap-1 group cursor-pointer'>
```
Comment (Odile078) [inline components/ui/LinkItem.jsx]: use Link component from Next js instead of a div tag, because this is going to be a link

**Review by Odile078 — APPROVED** [review]
LGTM

**Review by alain-kubwayo — APPROVED** [review]
LGTM!

**Review by Lily-Lilian — APPROVED** [review]

**Pull request merged**